src/API.jl

```diff
@@ -1387,7 +1387,7 @@ function recall_precompile_state()
         if isfile(fpath)
             open(fpath) do io
                 try
-                    pkgspecs = deserialize(io)
+                    pkgspecs = deserialize(io)    # what type does this have?
```

Comment:
I think a vector of `PackageSpec`. @IanButterworth probably knows.

Reply:
Bump @IanButterworth

Reply:
Yes sorry, didn't get notified for some reason. It's always a `Vector{PackageSpec}`